model/Hierarchy.php

```diff
@@ -11,30 +15,28 @@ class Hierarchy extends DataExtension {
 	
 	protected $markingFilter;
 	
-	/**
-	 * @var Int
-	 */
+	/** @var int */
```

Comment:
Have we established a style for single-line comments? If not, might be worth discussing...